tests/spam_test.py

```diff
@@ -19,6 +23,50 @@ def test_spam_configuration(self):
         self.testbot.assertCommand('!plugin config SpammingAlert '
                                    '{\'MAX_LINES\': 10}',
                                    'configuration done')
+        self.testbot.assertCommand('!plugin config SpammingAlert',
+                                   '{\'MAX_LINES\': 10}')
+        self.testbot.assertCommand('!plugin config SpammingAlert '
+                                   '{\'MAX_LINES\': 20, \'MAX_MSG_LEN\': 200}',
```

Comment:
use `'{\'MAX_MSG_LEN\':1, \'MAX_LINES\': 1}',` from the original code, so that the diff is aligned properly.

Reply:
@jayvdb this sets `MAX_MSG_LEN = 1`. So when the command msg `!plugin config SpammingAlert` is sent to the bot in the next line. The takes the cmd msg as a spam since MAX_MSG_LEN is set to 1.